styleguide/source/_patterns/05-pages/press.md

```diff
@@ -0,0 +1,4 @@
+### Description

```

Comment:
Update the title here as "Press Release" I worry that just "Press" will end up being confusing in the future or for content generators.